clang/test/Modules/relative-resource-dir.m

```diff
@@ -8,5 +8,6 @@
 // RUN:   -fimplicit-module-maps -fmodules-cache-path=%t.mcp \
 // RUN:   -fbuiltin-headers-in-system-modules \
 // RUN:   -resource-dir resource-dir \
+// RUN:   -internal-isystem resource-dir/include \
```

Comment:
I guess not all target triples add this path automatically, but without it this test fails for me even if I get rid of the <float.h> parts.